.github/issue_template.md

```diff
@@ -0,0 +1,48 @@
+Note: This is just a template, so feel free to use/remove the unnecessary things
+
+### Description
+- Type: Bug | Enhancement | Question
+- Related Issue: `#abc`
+- Priority: Blocker | Major | Minor
+
+---------------------------------------------------------------
+## Bug
+
+**target**
+K64F|??
+
+**toolchain:**
+gcc|armcc|IAR
```

Comment:
Nitpick: can we change this line to `GCC_ARM|ARM|IAR` to better reflect what we use in the tools and mbed-cli?